sops.go

```diff
@@ -0,0 +1,259 @@
+package sops
```

Comment:
I've been thinking about the directory structure some more, and I think it would make sense to put the main package at the root and the sops package under `/sops`.
That way, people who just want to cli, which is the majority anyway, would type `go get go.mozilla.org/sops`. And people who want the lib would import `go.mozilla.org/sops/sops`.
We can discuss it if you think it's a horrible idea.